src/tools/cephfs/cephfs-shell

```diff
@@ -1122,35 +1144,47 @@ sub-directories, files')
 
     du_parser = argparse.ArgumentParser(
         description='Disk Usage of a Directory')
-    du_parser.add_argument('dirs', type=str, action=path_to_bytes,
-                           help='Name of the directory.', nargs='?',
-                           default='.')
+    du_parser.add_argument('paths', type=str, action=get_list_of_bytes_path,
+                           help='Name of the directory.', nargs='*',
+                           default=[b'.'])
```

Comment:
No need for conversion to list, with nargs='*' the arguments passed are added to list. 

```suggestion
                           default=b'.')
```